Libraries/Components/TabBarIOS/TabBarIOS.android.js

```diff
@@ -12,11 +12,21 @@
 
 const React = require('React');
 const StyleSheet = require('StyleSheet');
-const TabBarItemIOS = require('TabBarItemIOS');
 const View = require('View');
 
+let showedDeprecationWarning = false;
+
 class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
-  static Item = TabBarItemIOS;
```

Comment:
Removing this caused Flow errors on Android:

```
Error ---------------------------------------------------------------------------
react-native/RNTester/js/TabBarIOSBarStyleExample.js:40:10
Cannot get `TabBarIOS.Item` because property `Item` is missing in statics of `DummyTabBarIOS` [1].
  react-native/RNTester/js/TabBarIOSBarStyleExample.js:40:10
  40|         <TabBarIOS.Item title="Tab" icon={{uri: base64Icon, scale: 3}} selected>
               ^^^^^^^^^^^^^^
References:
  react-native/Libraries/Components/TabBarIOS/TabBarIOS.android.js:19:7
  19| class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
            ^^^^^^^^^^^^^^ [1]
Error -------- react-native/RNTester/js/TabBarIOSExample.js:49:10
Cannot get `TabBarIOS.Item` because property `Item` is missing in statics of `DummyTabBarIOS` [1].
  react-native/RNTester/js/TabBarIOSExample.js:49:10
  49|         <TabBarIOS.Item
               ^^^^^^^^^^^^^^
References:
  react-native/Libraries/Components/TabBarIOS/TabBarIOS.android.js:19:7
  19| class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
            ^^^^^^^^^^^^^^ [1]
Error -------- react-native/RNTester/js/TabBarIOSExample.js:60:10
Cannot get `TabBarIOS.Item` because property `Item` is missing in statics of `DummyTabBarIOS` [1].
  react-native/RNTester/js/TabBarIOSExample.js:60:10
  60|         <TabBarIOS.Item
               ^^^^^^^^^^^^^^
References:
  react-native/Libraries/Components/TabBarIOS/TabBarIOS.android.js:19:7
  19| class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
            ^^^^^^^^^^^^^^ [1]
Error -------- react-native/RNTester/js/TabBarIOSExample.js:73:10
Cannot get `TabBarIOS.Item` because property `Item` is missing in statics of `DummyTabBarIOS` [1].
  react-native/RNTester/js/TabBarIOSExample.js:73:10
  73|         <TabBarIOS.Item
               ^^^^^^^^^^^^^^
References:
  react-native/Libraries/Components/TabBarIOS/TabBarIOS.android.js:19:7
  19| class DummyTabBarIOS extends React.Component<$FlowFixMeProps> {
            ^^^^^^^^^^^^^^ [1]
Found 4 errors
```

I'll fix it